css/style.css

```diff
@@ -0,0 +1,36 @@
+/* #image{
+    margin: auto;
+    max-width:fit-content;
+    max-height: fit-content;
+    
+}
+body{
+    text-align: center;
+}
+li{
+    text-align: left;
+    list-style-position: inside;
+} */
+
```

Comment:
- [ ] Please don't leave any unused code as a comment.

and a reason for that is they can make your code very noisy and hard to read when the project gets bigger.